Tests/UnitTests/Core/Propagator/StepperTests.cpp

```diff
@@ -264,12 +264,12 @@ BOOST_AUTO_TEST_CASE(eigen_stepper_test) {
   // Step size modifies
   const std::string originalStepSize = esState.stepSize.toString();
 
-  es.setStepSize(esState, 1337.);
-  BOOST_CHECK_EQUAL(esState.previousStepSize, navDir * stepSize);
-  BOOST_CHECK_EQUAL(esState.stepSize.value(), 1337.);
+  es.setStepSize(esState, 17.);
```

Comment:
Same here: why is this needed?

Reply:
don't remember but I guess this is arbitrary. will try to revert